workflows/calling/svc.nf

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env nextflow
+
+// Create queue channels (consumable)
+// will need to split up into tumor channel and normal channel, use regex for this
+tumor_ch = Channel.fromPath("${params.outdir}/aligned/duplicate_marked/*_T_*.bam")
+tumor_ch_bai = Channel.fromPath("${params.outdir}/aligned/duplicate_marked/*_T_*.bam.bai")
```

Comment:
Requiring a specific naming pattern for input files is a little weird. It requires the user to rename files in order to use workflow